tests/erica_legacy/request_processing/test_grundsteuer_request_controller.py

```diff
@@ -21,7 +21,19 @@ def valid_grundsteuer_request_controller():
 
 
 class TestIsTestmerkerUsed:
-    def test_returns_true(self, valid_grundsteuer_request_controller):
+
+    def test_if_tax_id_number_test_tax_id_then_returns_true(self, valid_grundsteuer_request_controller):
+        valid_grundsteuer_request_controller.input_data.eigentuemer.person[0].steuer_id = "04452397687"
+        result = valid_grundsteuer_request_controller._is_testmerker_used()
+        assert result is True
+
+    def test_if_tax_id_number_no_test_tax_id_then_returns_false(self, valid_grundsteuer_request_controller):
+        valid_grundsteuer_request_controller.input_data.eigentuemer.person[0].steuer_id = "43865766025"
+        result = valid_grundsteuer_request_controller._is_testmerker_used()
+        assert result is False
+
+    def test_if_no_tax_id_number_then_returns_true(self, valid_grundsteuer_request_controller):
+        valid_grundsteuer_request_controller.input_data.eigentuemer.person[0].steuer_id = None
         result = valid_grundsteuer_request_controller._is_testmerker_used()
         assert result is True
```

Comment:
About this I'm not so sure...is there ever a case this could happen? I think none that makes sense. So, in that case we'd want to get notified I think? Therefore, I'd probably instead throw an error. Returning True still is better than False of course, but maybe an error would make even more sense?

Reply:
I have added a separate validator that ensures that we always have a person. However, the steuer_id is an optional field. Therefore, we do not know if it is set. We talked about it and we'll keep it like that.